Readme.md

```diff
@@ -1,76 +1,25 @@
-
-The more methods you put in your package, the better.
-
-*Deployment*:
-
-Videos and screenshots will not be accepted. You must leave links to:
-
-1. Your NPM package
-2. A deployed project on Runkit or Codepen.
-
-
-Your deployment options for #2 above are:
-
-1. Runkit - You will have to sign up and create a new notebook. See the example [Runkit Notebooks](https://runkit.com/monarchwadia). 
-2. Codepen - You can import your project into Codepen using [unpkg](https://unpkg.com/)
-
-## Bonus points
-
-Bonus points will be awarded for:
-* Creating a typescript-compatible project
-* Creating unit tests
-* Using a precompiler (only if appropriate, i.e. if you are using Typescript or Babel, or if you are creating a React library)
-* Leaving detailed instructions for local development on `Readme.md`
-* Writing documentation (for your future self's benefit)
-
-## Frameworks:
-
-Any or none. You can create a personal library with just plain JavaScript. You can also create a personal library that contains common React/Vue/Angular components that you find yourself using often across projects.
-
-## Restrictions
-
-* Your project must use JavaScript.
-* You must follow deployment instructions above.
-
-There are no other restrictions.
-
-## Tips for success:
-
-1. The project is intentionally simple so you can focus on code quality.
-1. The requirements are intentionally minimalistic so you can get as creative as you'd like.
-1. You get a LOT of time for code review comments. So you can win even if you feel your project isn't great, simply on the strength of your code review comments!
-1. Keep in mind, 33% of your score is based on aesthetic appeal!
-
-## Resources
-
-* [Creating and publishing unscoped public packages](https://docs.npmjs.com/creating-and-publishing-unscoped-public-packages)
-* [Video - Creating & publishing your own NPM modules](https://www.youtube.com/watch?v=rTsz09zRuTU)
-* [freeCodeCamp's Tutorial](https://www.freecodecamp.org/news/how-to-make-a-beautiful-tiny-npm-package-and-publish-it-2881d4307f78/)
+# Ashley Claiborne's Utilities NPM Library
+
+## Install
+```
+$ npm install @aclai4067/utilities
+```
+
+## Usage
+> printToDom - reusable code for building html from your javascript file
+> Pass in an element id where you would like to print, then a variable name which holds html to print
+```
+printToDom(div1, htmlStringVariable);
+```
+
+> arrayToList - loops through an array that does not contain objects and prints each entry as an li element
+> Pass in your array name, then an id for your target UL or OL element
+
+```
+arrayToList(arrayVariable, ulId1);
+```
+
+> usCurrencyFormatter - converts a number to US currency format
+```
+usCurrencyFormatter.format(18793);
+```
```

Comment:
I love your Readme! It's very informative. One thing you could add to the description is why you created the library, well I guess that's kinda outlined in the Usage tab.